reana_server/rest/users.py

```diff
@@ -339,7 +339,8 @@ def get_me():
                     if me.latest_access_token else None,
                 },
                 'full_name': me.full_name,
-                'username': me.username}), 200)
+                'username': me.username,
+                'announcement': REANA_UI_ANNOUNCEMENT}), 200)
```

Comment:
I'm not sure if this is the best place to put the announcement. Perhaps we should think of another way to pass configuration to the UI in the future.

Reply:
Maybe we can open an issue describing it and once we have another use case we make a new endpoint for config?